src/components/icon/README.md

```diff
@@ -0,0 +1,116 @@
+# md-icon
+
+`md-icon` is a component that displays an icon, which can be a font glyph or an SVG image.
+
+## MdIconRegistry
+
+`MdIconRegistry` is an injectable service that allows you to associate icon names with SVG URLs and define aliases for CSS font classes. Its methods are discussed below and listed in the API summary.
+
+## Font icons
+
+### Ligatures
+
+Some fonts are designed to show icons by using [ligatures](https://en.wikipedia.org/wiki/Typographic_ligature), for example by rendering the text "home" as a home image. To use a ligature icon, put its text in the content of the `md-icon` component, for example:
+```html
+<md-icon>home</md-icon>
+```
+
+By default the [Material icons font](http://google.github.io/material-design-icons/#icon-font-for-the-web) is used. (You will still need to include the HTML to load the font and its CSS, as described in the link). You can specify a different font by setting the `fontSet` input to either the CSS class to apply to use the desired font, or to an alias previously registered with `MdIconRegistry.registerFontClassAlias`, for example:
+```javascript
+mdIconRegistry.registerFontClassAlias('myfont', 'my-icon-font-class');
```

Comment:
FontAwesome doesn't use ligatures as far as I can tell so it might be confusing. Changed the CSS font icon example below.